kvdb/postgres/db.go

```diff
@@ -256,5 +256,7 @@ func (db *db) Copy(w io.Writer) error {
 // Close cleanly shuts down the database and syncs all data.
 // This function is part of the walletdb.Db interface implementation.
 func (db *db) Close() error {
+	log.Infof("Closing database %v", db.prefix)
```

Comment:
Alternative fix could be to make `db.Close()` reference counted (last once to call will close).

Reply:
That is actually already happening in the `connSet`. That is also what caught this problem.

Reply:
Indeed, forgot that.